extensions/Cake.UrlLoadDirective.Module.yml

```diff
@@ -6,7 +6,7 @@ Assemblies:
 Repository: https://github.com/cake-contrib/Cake.UrlLoadDirective.Module.git
 ProjectUrl: https://github.com/cake-contrib/Cake.UrlLoadDirective.Module/
 Author: Redth
-Description: Cake module for supporting downloading files from url's in the #load directive.
```

Comment:
I have a feeling ` #load directive.` was removed because YamlDotNet (which the AddinDiscoverer uses to read existing YAML files) interprets it as a comment.

Reply:
So needs to be quoted?

Reply:
I can confirm it is the case. Description needs to be in quotes...

```yaml
Description: "Cake module for supporting downloading files from url's in the #load directive."
```